stressor/stress-test.mts

```diff
@@ -323,79 +408,179 @@ console.log(
 
 const server = http.createServer();
 server.listen(port);
-console.log(`Local server started at port ${port}`);
+debugLog(`Local server started at port ${port}`);
 server.setMaxListeners(50);
 
 const ngrokUrl = await ngrok.connect({
   port,
 });
-console.log(`Ngrok tunnel started at ${ngrokUrl}`);
+debugLog(`Ngrok tunnel started at ${ngrokUrl}`);
 
 process.on("beforeExit", (code) => {
   server.close();
   ngrok.kill();
-  console.log("Exiting with code: ", code);
+  console.error("Exiting with code: ", code);
 });
 
 const octokitClient = new Octokit({
   auth: process.env.GITHUB_TOKEN,
 });
 
-// Step through testPlans, waiting for those CI runs to finish before the next begin
-for (const testPlan of testPlans) {
-  console.log(`# Test plan: ${testPlan}\n`);
-  // Filter the list of test combos to only those for this test plan
-  const testCombosForTestPlan = testCombinations.filter(
-    (testCombo) => testCombo.workflowTestPlan === testPlan
-  );
-  // For each test plan, run each test combo in parallel
-  const testCombinationResults = await Promise.all(
-    testCombosForTestPlan.map(async (testCombo: TestCombination) => {
-      const runPromises = [];
-      for (let runIndex = 0; runIndex < numRuns; runIndex++) {
-        const dispatched = await dispatchWorkflowForTestCombo(
-          testCombo,
-          runIndex
-        );
-        if (dispatched) {
-          const listenerPromise = setUpTestComboCallbackListener(
-            testCombo,
-            runIndex
-          );
-          runPromises.push(listenerPromise);
-        }
-      }
-      debugLog(
-        `Dispatched ${
-          runPromises.length
-        } workflow runs for combination ${testComboToString(testCombo)}.`
-      );
-
-      // Wait to get all results from parallel runs of the same test combo
-      const runResults = await Promise.all(runPromises);
-
-      // Check if all the results are good
-      console.log(
-        `## ${workflowIdAsLabel(testCombo.workflowId)} on ${
-          testCombo.workflowBrowser
-        }.`
-      );
-      const runResultStats = checkRunSetResults(runResults);
-
-      return { ...testCombo, ...runResultStats };
-    })
-  );
 
-  console.log(`\n## Summary\n`);
-  console.log(`| Screenreader | Browser | Runs | Unequal Responses (%) |`);
-  console.log(`| --- | --- | --- | --- |`);
-  testCombinationResults.forEach((result) => {
-    console.log(
-      `| ${workflowIdAsLabel(result.workflowId)} | ${
-        result.workflowBrowser
-      } | ${numRuns} | ${result.percentUnequal.toFixed(2)}% |`
+
+// const allResults: Map<TestCombination, CompleteTestComboRunResult> = new Map();
+
+// Debug helper: write the needed "allResults" for this run to a json file
+import { readFile } from "node:fs/promises";
+
+const allResults: Map<TestCombination, CompleteTestComboRunResult> = new Map(
+  JSON.parse(await readFile("stressor-run.json", "utf-8"))
+);
```

Comment:
```suggestion
const allResults: Map<TestCombination, CompleteTestComboRunResult> = new Map();

// Debug helper: write the needed "allResults" for this run to a json file
// import { readFile } from "node:fs/promises";
// const allResults: Map<TestCombination, CompleteTestComboRunResult> = new Map(
//  JSON.parse(await readFile("stressor-run.json", "utf-8"))
// );
```

This might also want to be a feature of a command line flag, or process.env